esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6571,7 +6581,10 @@ bool CWsDfuEx::onDFUFilePublish(IEspContext &context, IEspDFUFilePublishRequest
 
         newFile->rename(newFileName, userDesc);
 
-        LOG(daliAuditLogCat,",FileAccess,EspProcess,CREATED,%s,%s,%s", groupName, userId.str(), newFileName.str());
+        StringBuffer auditUser;
+        createAuditUserString(context, userId, auditUser);
+        LOG(daliAuditLogCat,",FileAccess,Esp,%s,CREATE,%s@%s,,%s,\"method=WsDfu.DFUFilePublish,LockTimeoutMS=%d\"",
```

Comment:
does the audituser already contain the '@' ip? If not, should it be processed by createAuditUserString()?

Reply:
does anywhere contain @ , followed by an IP? The bit after the @ here is a group name..